rllib/examples/connectors/connector_v2_nested_action_spaces.py

```diff
@@ -0,0 +1,94 @@
+from gymnasium.spaces import Dict, Tuple, Box, Discrete, MultiDiscrete
+
+from ray.tune.registry import register_env
+from ray.rllib.connectors.env_to_module import (
+    AddObservationsFromEpisodesToBatch,
+    FlattenObservations,
+    WriteObservationsToEpisodes,
+)
+from ray.rllib.examples.env.multi_agent import MultiAgentNestedSpaceRepeatAfterMeEnv
+from ray.rllib.examples.env.nested_space_repeat_after_me_env import (
+    NestedSpaceRepeatAfterMeEnv,
+)
+from ray.rllib.utils.test_utils import (
+    add_rllib_example_script_args,
+    run_rllib_example_script_experiment,
+)
+from ray.tune.registry import get_trainable_cls
+
+
+# Read in common example script command line arguments.
+parser = add_rllib_example_script_args(default_timesteps=200000, default_reward=-500.0)
+
+
+if __name__ == "__main__":
+    args = parser.parse_args()
+
+    # Define env-to-module-connector pipeline for the new stack.
+    def _env_to_module_pipeline(env):
+        return [
+            AddObservationsFromEpisodesToBatch(),
```

Comment:
Yeah, I don't like this "cluster" of required connector pieces either. We need to have some easier way to do these 3 steps together: read-from-episode, change, write-back-to-episode. This is NOT an expensive operation, so it would be ok if the user had several such pieces, each of which does these in itself.